bundles/org.openhab.binding.matter/src/main/java/org/openhab/binding/matter/internal/util/MatterLabelUtils.java

```diff
@@ -58,7 +62,7 @@ public static String labelForNode(Endpoint root) {
         if (!nodeLabel.isEmpty()) {
             label = nodeLabel;
         } else {
-            label = vendorName + " " + productName;
+            label = productName.startsWith(vendorName) ? productName : vendorName + " " + productName;
```

Comment:
👍 I noticed yesterday  that Apple Home does this as well, so if i add "Office Lights" to a room called "Office", it changes the name of the device to just "Lights" when in that group.